genie-proto/src/main/proto/genie.proto

```diff
@@ -213,3 +213,15 @@ message ChangeJobStatusResponse {
     bool successful = 1;
     ChangeJobStatusError error = 2;
 }
+
+service JobKillService {
+    rpc registerAgent (JobKillListenerRequest) returns (JobKillListenerResponse);
+}
+
+message JobKillListenerRequest {
+    string job_id = 1;
+}
+
+message JobKillListenerResponse {
+    string job_id = 1;
```

Comment:
Did'nt have it initially. Thought of adding that since the agent should ideally check that the notification is indeed for itself before executing the kill. Just a safeguard mechanism.

Reply:
`JobKillListenerRequest` -> `JobKillRequest`
`JobKillListenerResponse` -> `JobKillResponse`

Reply:
Will do